homeassistant/components/light/wink.py

```diff
@@ -40,13 +45,20 @@ class WinkLight(WinkToggleDevice):
     # pylint: disable=too-few-public-methods
     def turn_on(self, **kwargs):
         """ Turns the switch on. """
+
+        temp_kelvin = None
+
         brightness = kwargs.get(ATTR_BRIGHTNESS)
 
+        if ATTR_COLOR_TEMP in kwargs:
+            temp_kelvin = int(((TEMP_MAX - TEMP_MIN) *
```

Comment:
Home Assistant is using Mireds for light units. According to [wikipedia](https://en.wikipedia.org/wiki/Mired) your formula cannot be used to convert from mireds to kelvin.

Please add two methods to [`util/color.py`](https://github.com/balloob/home-assistant/blob/dev/homeassistant/util/color.py) to convert between `mired` and `kelvin` and use those.